### PR TITLE
Isolate tests

### LINKED ISSUE #2: Isolate session on disk between test cases
This prevent side-effects between test cases via session that might
or might not pre-exist on disk.

As discussed in https://github.com/zendframework/zend-expressive-session-ext/pull/44#discussion_r330647766. Once that other PR is merged, I'll rebase this one to remove that `time()` usage that won't be necessary anymore.

Do test improvements also require a CHANGELOG entry ?

Provide a narrative description of what you are trying to accomplish:

- [ ] Are you fixing a bug?
  - [ ] Detail how the bug is invoked currently.
  - [ ] Detail the original, incorrect behavior.
  - [ ] Detail the new, expected behavior.
  - [ ] Base your feature on the `master` branch, and submit against that branch.
  - [ ] Add a regression test that demonstrates the bug, and proves the fix.
  - [ ] Add a `CHANGELOG.md` entry for the fix.

- [ ] Are you creating a new feature?
  - [ ] Why is the new feature needed? What purpose does it serve?
  - [ ] How will users use the new feature?
  - [ ] Base your feature on the `develop` branch, and submit against that branch.
  - [ ] Add only one feature per pull request; split multiple features over multiple pull requests
  - [ ] Add tests for the new feature.
  - [ ] Add documentation for the new feature.
  - [ ] Add a `CHANGELOG.md` entry for the new feature.

- [x] Is this related to quality assurance?
  

- [ ] Is this related to documentation?
  
  

---

Originally posted by @PowerKiKi at https://github.com/zendframework/zend-expressive-session-ext/pull/46

### PR DESCRIPTION
**Pull request by PowerKiKi**
|    Q        |   A
|------------ | ------
| Bugfix      | no
| BC Break    | no
| New Feature | no
| RFC         | no
| QA         | yes

### Description

This is a cherry/pick of https://github.com/zendframework/zend-expressive-session-ext/pull/46 to fix https://github.com/mezzio/mezzio-session-ext/issues/2

### CODE REVIEW

File: test/PhpSessionPersistenceTest.php [hunk 1]
```diff
@@ -46,11 +46,46 @@ class PhpSessionPersistenceTest extends TestCase
...
+    private $originalSessionSettings;
+
+    /**
+     * @var string
+     */
+    private $sessionSavePath;
+
     public function setUp()
     {
+        $this->sessionSavePath = sys_get_temp_dir() . "/zend-expressive-session-ext";
```
Comment (michalbundyra) [inline test/PhpSessionPersistenceTest.php]: Please use `mezzio` instead of `zend-expressive`.

**Review by michalbundyra — APPROVED** [review]
👍

File: test/PhpSessionPersistenceTest.php (hunk 1 above)
Comment (PowerKiKi) [reply test/PhpSessionPersistenceTest.php]: Fixed

**Comment by michalbundyra** [comment]
Thanks, @PowerKiKi!

**Pull request merged**